### PR TITLE
Call change.on_remove when deleting an running action

### PR DESCRIPTION
**Pull request by ferbncode**
Improvements made: 

* Removing throttle configurations if action is deleted
* Checking that throttle limit is set such that  `throttle > max(BytesIn)`
* Add command `remove-throttle` to remove throttle config from all brokers and topics.

### CODE REVIEW

File: bubuku/cli.py [hunk 1]
```diff
@@ -103,11 +102,15 @@ def rolling_restart_broker(image_tag: str, instance_type: str, scalyr_key: str,
...
-
 @click.option('--throttle', type=click.INT, default=100000000, help="Upper bound on bandwidth (in bytes/sec) used for "
                                                                     "rebalance")
 def rebalance_partitions(broker: str, empty_brokers: str, exclude_topics: str, parallelism: int, bin_packing: bool,
                          throttle: int):
+
+    if throttle and throttle < get_max_bytes_in():
+        print('Throttle value must be set above the max BytesIn for the replication to progress. '
+              'The current max BytesIn is {}'.format(get_max_bytes_in()))
+        return
```
Comment (antban) [inline bubuku/cli.py]: can you please do exit(1) so calling process can have some info about unsuccessful? result

**Comment by antban** [comment]
:+1:

**Comment by ferbncode** [comment]
:+1:

**Comment by adyach** [comment]
👍

**Pull request merged**